universe/interface.go

```diff
@@ -50,6 +50,19 @@ func (i *Identifier) String() string {
 	return hex.EncodeToString(i.AssetID[:])
 }
 
+// StringForLog returns a string representation of the ID for logging.
+func (i *Identifier) StringForLog() string {
+	groupKey := "<nil>"
+	if i.GroupKey != nil {
+		groupKey = hex.EncodeToString(
+			schnorr.SerializePubKey(i.GroupKey),
+		)
+	}
+
+	return fmt.Sprintf("%v (asset_id=%x,group_key=%v)",
+		i.String(), i.AssetID[:], groupKey)
+}
```

Comment:
I like JSON as well, but I don't want to start mixing formats, as we normally use `spew.Sdump()` in the code base. Here that isn't sufficient because we have defined a `String()` method that doesn't show all information (but `spew.Sdump()` always uses that method if it exists).